overviews/quasiquotes/expression-details.md

```diff
@@ -75,20 +75,20 @@ Each term identifier is defined by its name and by the fact of being backquoted
     scala> val backquoted = q"`$name`"
     backquoted: universe.Ident = `Foo`
 
-Although backquoted and non-backquoted identifiers may refer to the same things they are not equivalent from syntactical point of view:
+Although backquoted and non-backquoted identifiers may refer to the same thing they are not equivalent from a syntactical point of view:
 
     scala> val q"`Foo`" = q"Foo"
     scala.MatchError: Foo (of class scala.reflect.internal.Trees$Ident)
       ... 32 elided
 
-This is caused by the fact that backquoted identifiers have different semantics in pattern patching.
+This is because backquoted identifiers have different semantics in pattern patching.
 
-Apart from matching on identifiers with given name you can also extract their name values with the help of [unlifting](/overviews/quasiquotes/unlifting.html):
+Apart from matching on identifiers with a given name, you can also extract their name values with the help of [unlifting](/overviews/quasiquotes/unlifting.html):
 
     scala> val q"${name: TermName}" = q"Foo"
     name: universe.TermName = Foo
 
-Name ascription is important here as without it you\'ll get pattern that is equivalent to regular pattern variable binding.
+Name ascription is important here because without it you\'ll get a pattern that is equivalent to regular pattern variable binding.
```

Comment:
throughout the document, I don't think putting backslash before single quote is necessary